apps/camera/style/recording-timer.css

```diff
@@ -19,7 +20,7 @@
   text-align: left;
   pointer-events: none;
   opacity: 0;
-  transition: opacity 0.2s ease-in-out;
+  transition: opacity 0.2s ease-in-out, visibility 0s ease-in-out 0.2s;
```

Comment:
nit: same here -- I don't think its necessary to specify an easing function for `visibility`